providers/implementations/signature/ecdsa.c

```diff
@@ -82,7 +84,7 @@ typedef struct {
      */
     BIGNUM *kinv;
     BIGNUM *r;
-#if defined(FIPS_MODULE) && !defined(OPENSSL_NO_ACVP_TESTS)
```

Comment:
Why are we dropping FIPS_MODULE here?

Reply:
I am trying to reduce the FIPS_MODULE dependencies here..
It is not worth the hassle for a single variable..
This ACVP_TESTS define should only be used by the labs..